Tests/EventListener/RequestExecutedListenerTest.php

```diff
@@ -19,17 +19,15 @@
  * @copyright 2018 Chaplean (http://www.chaplean.coop)
```

Comment:
http -> https everywhere if possible (not your fault but if you make a new commit before merging ;))